site/_docs/reference.md

```diff
@@ -2359,6 +2359,7 @@ semantics.
 | m | SPACE(integer)                                 | Returns a string of *integer* spaces; returns an empty string if *integer* is less than 1
 | o | SUBSTR(string, position [, substringLength ]) | Returns a portion of *string*, beginning at character *position*, *substringLength* characters long. SUBSTR calculates lengths using characters as defined by the input character set
 | m | STRCMP(string, string)                         | Returns 0 if both of the strings are same and returns -1 when the first argument is smaller than the second and 1 when the second one is smaller the first one.
+| m | INSTR(string, string)                          | Returns the index of the first occurrence of the second argument in the first argument and returns 0 if the first argument not include the second argument.
```

Comment:
We'd better remove the period at the end of the description.

Reply:
well, STRCMP does the same, change it altogether